toolchains/lowrisc_toolchain_rv32imc/lowrisc_toolchain_rv32imc.bzl

```diff
@@ -135,7 +135,7 @@ def _lowrisc_toolchain_rv32imc_toolchain_config_info_impl(ctx):
         sysroot = None,
     )
     embedded_features = GetEmbeddedFeatures(
-        compiler = "GCC",
+        compiler = "CLANG",
```

Comment:
Shouldn't line 152 below (`compiler = "riscv32-unknown-elf-gcc"`) be `...-clang`?

Reply:
Thanks, Sorry I missed that.